lib/src/base-styles/_nx-btn.scss

```diff
@@ -79,24 +79,22 @@
 }
 
 .nx-btn--error {
-  background-color: var(--nx-swatch-red-45);
-  border-color: var(--nx-swatch-red-45);
+  background-color: var(--nx-swatch-red-40);
+  border-color: transparent;
   color: var(--nx-swatch-white);
 
-  &:hover, &:focus, &:active {
-    background-color: var(--nx-swatch-red-30);
-    border-color: var(--nx-swatch-red-30);
-    color: var(--nx-swatch-white);
-  }
-
   &:focus {
-    background-color: var(--nx-swatch-red-45);
+    background-color: var(--nx-swatch-red-40);
```

Comment:
Can be removed, right?

Reply:
> Can be removed, right?

Ah right, yes it can: https://github.com/sonatype/sonatype-react-shared-components/pull/481/commits/5672f7696e727ba4d80bb51c8cc2676834b35edb